core/src/main/java/com/klaytn/caver/methods/response/GovernanceChainConfig.java

```diff
@@ -267,6 +273,12 @@ public static class Reward {
             @JsonProperty("ratio")
             private String ratio;
 
+            /**
+             * The value of kip82Ratio field.
+             */
+            @JsonProperty("kip82ratio")
```

Comment:
kip82ratio JsonProperty need to change to kip82Ratio also.

Reply:
The json name of the kip82Ratio is the "json82ratio". 
- ref. https://github.com/klaytn/klaytn/blob/dev/params/config.go#L207